A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -0,0 +1,53 @@
+class CDUAocOfpData {
+    static ShowPage(mcdu) {
+        mcdu.clearDisplay();
+        mcdu.page.Current = mcdu.page.AOCOfpData;
+        mcdu.activeSystem = 'ATSU';
+
+        let blockFuel = "_____[color]red";
+        let taxiFuel = "____[color]red";
+        let tripFuel = "_____[color]red";
+        let estZfw = "__._[color]red";
+
+        function formatWeight(value) {
+            return (+value / 1000).toFixed(1);
+        }
```

Comment:
I'm assuming this converts KG to Tons. extracting it to FMCMainDisplay could help others.

Reply:
Same as above. Maybe those methods can be used outside of the FMC scope, so Im not sure if its ideal to have all this funtions on this one big file

Reply:
https://github.com/flybywiresim/a32nx/pull/1865#discussion_r524276229

Reply:
why the `+`?
and why doing this as a asperate function?